### PR TITLE
Node version manager settings

### PR DESCRIPTION
**Pull request by maratfakhreev**
In my opinion we need to use NVM instead of the Node globally. There are base .bashrc/.zshrc settings for NVM, other changes I will put into osx-bootstrap

### CODE REVIEW

File: node/completion.bash [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export NVM_DIR=/usr/local/var/nvm
+
+nvm_completion="$(brew --prefix nvm)/nvm.sh"
```
Comment (timurvafin) [inline node/completion.bash]: @maratfakhreev looks like `nvm_completion` is incorrect name for this file b/c this file contains `nvm` actually.
And bash completion should be loaded in different way https://github.com/creationix/nvm#bash-completion

Comment (timurvafin) [inline node/completion.bash]: Please use following structure:
- nvm/config.bash - to load nmv
- nvm/completion.bash - to setup bash completion

Comment (timurvafin) [inline node/completion.bash]: @maratfakhreev ^^^

**Comment by maratfakhreev** [comment]
cc @timurvafin Sorry for the late with fix and thanks for remarks

**Comment by timurvafin** [comment]
@maratfakhreev please make sure `script/ci` works without warnings.

**Comment by maratfakhreev** [comment]
@timurvafin Done.

**Pull request merged**